internal/testhelper/ci-ar-cleanup-policy.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
I put this json file in testhelper because I think it's for ci tests.
If we think putting this file to other directories like `artifact-registry/ci-ar-cleanup-policy.json` or echo this file in `ci workflow`, I think that's also doable.